### PR TITLE
add a origin_json_path variable to  function "_generate_test_data"（to master branch）

### PR DESCRIPTION
**Pull request by monkeywq**
Hello，I review the function _generate_test_data in test_vcompressor.py. I think it is better to add a variable to replace extern function get_tests_data_file_path. After that the code become easier to read and reduce the number of function calls.
its my first time to create pull request.Please tell me how to correct it if I make many mistakes，Thanks！（Sorry, I changed the merage branch to master）

### CODE REVIEW

File: tests/test_vcompressor.py [hunk 1]
```diff
@@ -464,16 +464,17 @@ def _generate_test_data(self, test_file):
         with tempfile.TemporaryDirectory() as tmpdir:
             cvf_path = os.path.join(tmpdir, "result.cvf")
             dup_json_path = os.path.join(tmpdir, "result.json")
+            origin_json_path=get_tests_data_file_path(test_file)
```
Comment (gaogaotiantian) [inline tests/test_vcompressor.py:467]: Sure, just add spaces around the `=`.

Comment (monkeywq) [reply tests/test_vcompressor.py:467]: OK，should I commit another PR after modify?

Comment (gaogaotiantian) [reply tests/test_vcompressor.py:467]: No you should just push to this branch, the PR will be automatically updated.

Comment (monkeywq) [reply tests/test_vcompressor.py:467]: Sure, it seems to be something wrong on my internet,I will push it later,Thanks!

Comment (monkeywq) [reply tests/test_vcompressor.py]: I committed it on the github online editor,can you review it?

**Pull request merged**